cpp/src/dictionary/add_keys.cu

```diff
@@ -55,6 +55,7 @@ std::unique_ptr<column> add_keys(
   CUDF_EXPECTS(new_keys.type() == old_keys.type(), "Keys must be the same type");
   // first, concatenate the keys together
   // [a,b,c,d,f] + [d,b,e] = [a,b,c,d,f,d,b,e]
+  auto keys_vector = std::vector<column_view>{old_keys, new_keys};
```

Comment:
what is this change for?

Reply:
You can't pass an init list to a function expecting a `host_span` like you can to a function expecting a `vector`.  So you have to create the vector expicitly.

Reply:
There was a temporary change due to problems with span that required explicitly invoking the span ctor though, and this line got left in. Thanks for catching.